recipes/sip/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "4.18" %}
+
+package:
+  name: sip
+  version: {{ version }}
+
+source:
+  fn:   sip-{{ version }}.tar.gz  # [unix]
+  url:  http://sourceforge.net/projects/pyqt/files/sip/sip-{{ version }}/sip-{{ version }}.tar.gz  # [unix]
+  sha1: 6704854bc684de3b76c9db61ce54a74e7de5cf45  # [unix]
+  fn:   sip-{{ version }}.zip  # [win]
+  url:  http://sourceforge.net/projects/pyqt/files/sip/sip-{{ version }}/sip-{{ version }}.zip  # [win]
+  sha1: afde507fbee85a1aeb5fcb32e897bd3ea2bc48f5  # [win]
```

Comment:
This is pretty subtle, but is there actually something different between the two sources? I know they do provide one for Windows and one for Mac and Linux, but they do look pretty similar when one downloads them. I remember discussion about line endings being different that @groutr has raised, but in discussion with @msarahan we determined this shouldn't matter. Not a big deal in any event, but figured I should figure out if there was something else I was missing here.